src/main/python/model/postbuilder.py

```diff
@@ -81,12 +93,18 @@ def __get_file_name(self, metadata):
             save_name += f" ({metadata['beq_source']})"
         if self.__selected_signal is not None and not math.isclose(self.__selected_signal.offset, 0.0):
             save_name += f' ({self.__selected_signal.offset:+.1f} gain)'
-            metadata['beq_gain'] = f'{self.__selected_signal.offset:+.1f}'
+            metadata['beq_gain'] = f'{self.__selected_signal.offset:+.1f} gain'
         if len(metadata["beq_audioTypes"]) > 0:
             save_name += f' BEQ {metadata["beq_audioTypes"][0]}'
         save_name = save_name.replace(':', '-')
         return save_name
 
+    def autofillSortTitle(self):
+        if self.sortField.text().strip() == '':
+            title = self.titleField.text().strip()
+            title = title[title.startswith("The ") and len("The "):]
```

Comment:
case insensitive perhaps?

Reply:
Not a bad idea for more general use.  I had it case sensitive in mine since I'm used to entering the "correct" way for it to work. :)